tests/components/squeezebox/conftest.py

```diff
@@ -221,6 +221,7 @@ def mock_pysqueezebox_player(uuid: str) -> MagicMock:
         mock_player.remote_title = None
         mock_player.title = None
         mock_player.image_url = None
+        mock_player.model = "Squeezelite"
```

Comment:
FYI, this says `Squeezelite` while we check for `SqueezeLite`, what should it be?

Reply:
Yes - SqueezeLite is correct.  Test didn't seem to change, but have updated.